client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -102,28 +137,29 @@ const DomainPicker: FunctionComponent< Props > = ( {
 					{ suggestions?.length
 						? suggestions.map( suggestion => (
 								<Button
-									onClick={ () => onDomainSelect( suggestion ) }
+									onClick={ () => handleDomainClick( suggestion ) }
 									className="domain-picker__suggestion-item"
 									key={ suggestion.domain_name }
 								>
 									<span className="domain-picker__suggestion-item-name">
 										{ suggestion.domain_name }
 									</span>
-									{ suggestion.is_free ? (
-										<span className="domain-picker__suggestion-action">{ NO__( 'Select' ) }</span>
-									) : (
-										<a
-											className="domain-picker__suggestion-action"
-											href={ `http://wordpress.com/start/domain?new=${ suggestion.domain_name }` }
-											target="_blank"
-											rel="noopener noreferrer"
-										>
-											{ NO__( 'Upgrade' ) }
-										</a>
-									) }
+									<span className="domain-picker__suggestion-action">
+										{ suggestion.is_free ? (
+											<span>
+												<span className="domain-picker__price">{ NO__( 'Free' ) }</span>
+												{ NO__( 'Select' ) }
+											</span>
+										) : (
+											<span>
+												<span className="domain-picker__price">{ NO__( 'from €4/month' ) }</span>
```

Comment:
No idea how to localize or parametrize this `€4` bit here, but pretty sure we can't send this to translating as-is. Might be fine to leave as-is and just leave an inline todo?

Reply:
It took me a minute to get this. We need to know the localized price of our cheapest plan that includes a domain here, right?

That's going to be tricky… we might need to migrade plan data to a data store 😬

Reply:
For localizing currencies, we have a very robust solution on the way: https://github.com/Automattic/wp-calypso/pull/36758 and a lesser solution already exists: https://github.com/Automattic/wp-calypso/tree/master/packages/format-currency.

Neither of those is probably relevant, I think plan data is already localized. Those libraries also don't take into consideration any adjustment for different currency values, they're strictly formatting.

Reply:
We're probably better off just changing the copy. 😁

Reply:
I agree!